src/mousev1_to_nwb/abdeladim_2023/abdeladim_2023imaginginterface.py

```diff
@@ -0,0 +1,128 @@
+from dateutil.parser import parse as dateparse
+import datetime
+import json
+from pathlib import Path
+from natsort import natsorted
+from .abdeladim_2023imagingextractor import (
+    Abdeladim2023MultiPlaneImagingExtractor,
+    Abdeladim2023SinglePlaneImagingExtractor,
+)
+from neuroconv.datainterfaces.ophys.baseimagingextractorinterface import BaseImagingExtractorInterface
+from neuroconv.utils import FolderPathType
+from neuroconv.utils.dict import DeepDict
+from roiextractors.extractors.tiffimagingextractors.scanimagetiff_utils import extract_extra_metadata
+from roiextractors.extractors.tiffimagingextractors.scanimagetiffimagingextractor import ScanImageTiffSinglePlaneImagingExtractor
+
+
+class Abdeladim2023SinglePlaneImagingInterface(BaseImagingExtractorInterface):
+    """
+    Data Interface for writing imaging data for the MouseV1 to NWB file using Abdeladim2023SinglePlaneImagingExtractor.
+    """
+
+    Extractor = Abdeladim2023SinglePlaneImagingExtractor
+
+    @classmethod
+    def get_available_channels(cls, folder_path):
+        file_path = sorted(glob.glob(f"{folder_path}/*.tif"))
```

Comment:
hmm I think I had a comment to use natsor.natsorted instead of sorted?

Reply:
Yes, it override the correction when I merged. I will also checked the other comments